app/views/finders/_results.mustache

```diff
@@ -23,8 +23,14 @@
 
 {{#zero_results}}
   <div class='no-results'>
-    <h2 class='gem-c-heading'>There are no matching results.</h2>
-    <p>Try making your search broader.</p>
+    <p class='govuk-body govuk-!-font-size-19 govuk-!-font-weight-bold'>There are no matching results.</p>
+    <p class='govuk-body govuk-!-font-size-19'>Improve your search results by:</p>
+    <ul class="govuk-list">
+      <li class='govuk-!-font-size-19'> - removing filters<li>
```

Comment:
These `<li>` tags need to be closed with `</li>`.

Reply:
fixes